test/feature/idl_parser/idl_extra_cases/bit_bound_annotation.idl

```diff
@@ -0,0 +1,29 @@
+enum bit_bound_ann_valid_enum
+{
+    @bit_bound(8) ENUM_VALUE_1,
```

Comment:
Annotation `@bit_bound` is only valid for `enum` and `bitmask` types. It is invalid in members